app/src/main/java/org/wikipedia/readinglist/ReadingListsFragment.kt

```diff
@@ -171,12 +190,34 @@ class ReadingListsFragment : Fragment(), SortReadingListsDialog.Callback, Readin
                 }).show()
         }
 
+        override fun importNewList() {
+            var filePickerIntent = Intent(Intent.ACTION_GET_CONTENT)
+            filePickerIntent.type = "application/json"
+            filePickerIntent = Intent.createChooser(filePickerIntent, getString(R.string.reading_lists_import_file_picker_title))
+            filePickerLauncher.launch(filePickerIntent)
+        }
+
+        override fun editList() {
```

Comment:
The name "edit list" does not make sense to me, because it is actually starting the multiselection mode. Would it be possible to change the name to "Select", just like what we have done in the `ReadingListFragment`?

Reply:
Will discuss this with Robin as well, as he has named the menu item as "Edit" 😅 . Will change both if he agrees.